### ISSUE TITLE
Remaining time tab never fully completes

### ISSUE THREAD

**Issue by felipemldias** [trgkanki/remaining_time#42]
# Bug report

## Describe the bug

> Review bar never fully completes, leaving a margin at the end

## Reproduction step

> I have experimented this with every deck, regardless of settings, Anki version and all other add-ons disabled

## Expected behavior

> Bar completes at the end of deck sequence

## Actual behavior

 ![image](https://user-images.githubusercontent.com/63819261/108002426-59794d00-6fce-11eb-92fa-2d76c0951608.png)

## Anki/OS versions

- OS: *[macOS Big Sur]*
- Anki version *2.1.40 and 2.1.35 (I updated it to see if that was the problem)*

## Additional context

> Add any other context about the problem here.

**Comment by felipemldias** [comment]
Dear author, first of all, thank you so much for this add-on. It helps immensely and this is just a minor detail I wanted to share.

**Comment by phu54321** [comment]
Thanks for the report. On my system, the progress bar completely disappears at the end of the review session.

![image](https://user-images.githubusercontent.com/775265/108153913-6a85a380-711f-11eb-8879-456e376d8f9b.png)

Will this behavior do? I don't yet know what caused progress bar to persist on your side and not on my side, but maybe I could just force this behavior.

**Comment by felipemldias** [comment]
The bar doesn't persists when I reopen a completed deck, only right after I finished the review session, which isn't really an issue. I just think it was nice to use the bar as a visual parameter of where I was in the review session, which isn't the case when the bar doesn't completes. The ETA and remaining time still fulfill that purpose, but it isn't as engaging. Anyway, I know this is only an aesthetic detail, and I really wouldn't want to waste your time. I tried downloading previous versions of the add-on from the Github repository to see if I could solve this on my own, but I don't know how to build them from the files 😅

**Comment by felipemldias** [comment]
PS: While I'm here, is there a shortcut to reset the bar? I tried "h" as per the comments on the ankiweb page, but it didn't work. Thank you so much once again!

**Comment by felipemldias** [comment]
Dear author,

Just giving some further feedback, since my last post I've tried the add-on in another mac and also tried changing the time-zone, but the issue persisted.
 
Also, I noticed that many times the color assigned in the bar doesn't correspond to my answers, with wrong answers coloring the bar red much later on in the review, which leads me to believe there may be a delay between the answers and their representation in the bar. Perhaps this is caused by buried cards, but I'm not sure. 

I also captured this video of the bar behaving a little erratically, with the remaining time and the bar length jumping suddenly.

https://user-images.githubusercontent.com/63819261/111038332-1b255100-8407-11eb-993f-a3f0a25a68b3.mov

PS: In this video, the colors for good are green and new are gray, as I felt these colors corresponded better to the answer buttons colors. It would be really cool to be able to label "hard" orange and "easy" blue.

Best regards , 

Felipe

**Comment by phu54321** [comment]
Sorry for late reply. ETA today.

**Comment by felipemldias** [comment]
Hey man, no worries, and thank you for taking a look! I'm not even sure my issue is reproducible, so maybe there's nothing that can be done about it.

**Comment by phu54321** [comment]
Could you try with this build?

[build.ankiaddon.zip](https://github.com/trgkanki/remaining_time/files/6162462/build.ankiaddon.zip)

**Comment by felipemldias** [comment]
Hey, I tried it with all other addons toggled off, with autoreset turned off and on, and also with debug turned off and on, on both anki 2.1.35 and 2.1.42 to no avail unfortunately.

I also noticed it sometimes adds a red segment at the start automatically, even though I answered correct on the whole deck, independently from which settings are on. This is because I download some small "test decks" from ankiweb in order to test the addon. 

![image](https://user-images.githubusercontent.com/63819261/111618310-fa704900-87c2-11eb-876d-1dbf244b40c2.png)
